### PR TITLE
Commons division atom

### PR DESCRIPTION
**Pull request by tomrf1**
I considered using the content-entity library to model the MPs, but decided against it. It would involve creating the entities in capi and linking to them with entityIds. We can add an optional Person entity field later if we decide it's useful to have that relationship.

### CODE REVIEW

**Comment by tsop14** [comment]
👍 I like the idea of adding entities and constituencies later.

File: thrift/src/main/thrift/atoms/commonsdivision.thrift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+namespace * contentatom.commonsdivision
+namespace java com.gu.contentatom.thrift.atom.commonsdivision
+#@namespace scala com.gu.contentatom.thrift.atom.commonsdivision
```
Comment (tsop14) [inline thrift/src/main/thrift/atoms/commonsdivision.thrift]: Why do we need this? @mchv do you know?

Comment (mchv) [reply thrift/src/main/thrift/atoms/commonsdivision.thrift]: no sorry

**Comment by mchv** [comment]
> We can add an optional Person entity field later if we decide it's useful to have that relationship.

Indeed that would be an nice enhancement in the future, allowing us to report automatically how a member of parliament has voted in the common, but agree it is easier to not do it right now

**Comment by tomrf1** [comment]
@mchv https://www.theyworkforyou.com is a nice implementation of that

**Pull request merged**